python/ray/parameter.py

```diff
@@ -69,6 +69,9 @@ class RayParams(object):
         autoscaling_config: path to autoscaling config file.
         _internal_config (str): JSON configuration for overriding
             RayConfig defaults. For testing purposes ONLY.
+        include_java(bool): If True, this cluster could enables cross-languages
+            invocation.
```

Comment:
`If True, the raylet backend can also support Java worker`. 
Note, this only enables running Python & Java simultaneously. Cross-language invocation hasn't been implemented yet.